src/components/MissionCard.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { handleMission } from '../redux/Missions/missions';
```

Comment:
You wrote the missions folder name with capital letter, which should be the small '../redux/missions/missions';'. the same error in Missions.js and store.js files.